src/Administration/Migration/V6_7/Migration1737472122TokenUser.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Administration\Migration\V6_7;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ */
+#[Package('after-sales')]
+class Migration1737472122TokenUser extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1737472122;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->executeStatement('
+            CREATE TABLE IF NOT EXISTS token_user (
+                `id` BINARY(16) NOT NULL,
+                `user_id` BINARY(16) NOT NULL,
+                `user_sub` VARCHAR(255) NOT NULL,
+                `refresh_token` TEXT NOT NULL,
+                `provider` VARCHAR(32) NOT NULL,
+                `expires` DATETIME NOT NULL,
```

Comment:
Small nitpick: `expiry` is often used for the timestamp (e.g. in the cookie). We also have `expires_in` which is the remaining time. Just `expires` could be confusing and it's better to stay consistent through the whole implementation :)

Reply:
I understand your point, but expires_in contains an int which means that "it expires in seconds" and in this case its a point where it expires. Maybe we could name it `expires_at`

Reply:
The cookie property name was `expiry`, and I think we should stick to that throughout the whole codebase, including the DB column for the timestamp.